HomeWork/src/main/java/exception/ArrayValueCalculator.java

```diff
@@ -0,0 +1,55 @@
+package exception;
+
+import string.StringLevelOne;
+
+import java.util.Arrays;
+import java.util.zip.DataFormatException;
+
+public class ArrayValueCalculator {
+
+    int results = 0;
+    String[][] str = new String[4][4];
+
+    public void arrayInitialization(String[][] strings1) {
+        for (int i = 0; i < strings1.length; i++) {
+            for (int j = 0; j < strings1[i].length; j++) {
+                strings1[i][j] = "5";
+            }
+        }
+    }
+
+    public int doCalc(String[][] strings1) {
+        if (strings1.length != str.length || strings1[0].length != str[0].length) {
```

Comment:
Just compare it like this `strings1.length != MAX_MATRIX_SIZE`, where `MAX_MATRIX_SIZE` is a constant: `private static final int MAX_MATRIX_SIZE = 4;`